CMSgov/onemac
# Oy2 175 cms access attachments

Changes:
1. Allow permanent read access to attachments.  Before this change, we could only obtain links that would expire after a max of 7 days.  This was a problem as CMS needs access to the attachments past the 7 days.  This change open public access to the attachments.
2. Switched the attachment upload to protected instead of private for a possible future with CMS receivers logging in to see the files.

Test:
1. Submit a new SPA or waiver with one or more attachments.  
2. Inspect the data in the database and verify that the URL for the attachments does not contain URL parameters (there is no ? or anything after.  Without parameters it means the link is a perma link.
3. Wait 15 mins or more and access the link from the database to verify the link does not expires.  15 mins is the default expiration when getting signed URLs.  This test is kind of overkill, but it proves the point.

Endpoint: https://d1w8lpmoz4ksr2.cloudfront.net/
